src/Forms/Shared/Samples/GraphicsOverlay/AddGraphicsRenderer/AddGraphicsRenderer.xaml.cs

```diff
@@ -179,5 +199,25 @@ private Geometry MakeHeartGeometry(MapPoint center, double sideLength)
             builder.AddPart(newPart);
             return builder.ToGeometry();
         }
+
+        private Polygon MakeEllipseGeometry()
+        {
+            // create parameters and set all the parameters
```

Comment:
For the public samples, we try to be very careful about always starting comments with a capital letter, and ending with a period.